spyder/plugins/editor/widgets/editor.py

```diff
@@ -694,6 +696,126 @@ def create_shortcuts(self):
                                       context="Editor",
                                       name="close split panel",
                                       parent=self)
+        save_bookmark_1 = config_shortcut(lambda:
+                                          self.sig_save_bookmark.emit(0),
+                                          context="Editor",
+                                          name="bookmark current position "
+                                               "to slot 0",
+                                          parent=self)

```

Comment:
This is very un-DRY code. Can you make these two into list comprehensions or for loops instead, and then just concatenate the resulting list to the one being returned at the end?

Reply:
This should be fixed now!